services/src/unit-test/java/uk/gov/moj/sdt/services/TargetApplicationSubmissionServiceTest.java

```diff
@@ -534,84 +608,75 @@ public void processRequestToSubmitSoapFault() {
-            }
-        });
-
-        this.mockIndividualRequestDao.persist(individualRequest);
-        EasyMock.expectLastCall();
+        doAnswer(invocation -> {
+            IndividualRequest argument = invocation.getArgument(0);
+            argument.setRequestStatus(IIndividualRequest.IndividualRequestStatus.ACCEPTED.getStatus());
+            return null;
+        }).when(mockConsumerGateway).individualRequest(individualRequest, 1000, 12000);
 
         final IBulkSubmission bulkSubmission = individualRequest.getBulkSubmission();
 
-        EasyMock.expect(
-                this.mockIndividualRequestDao.queryAsCount(EasyMock.same(IndividualRequest.class), EasyMock.isA(Supplier.class))).andReturn(0L);
-
-        mockIndividualRequestDao.persist(bulkSubmission);
-        EasyMock.expectLastCall();
-
-        EasyMock.replay(mockIndividualRequestDao);
-        EasyMock.replay(mockConsumerGateway);
-        EasyMock.replay(mockCacheable);
-        EasyMock.replay(mockErrorMsgCacheable);
+        when(
+             this.mockIndividualRequestDao.queryAsCount(eq(IndividualRequest.class),
+                                                        isA(Supplier.class))).thenReturn(0L);
 
         // Setup dummy response
         SdtContext.getContext().setRawInXml("response");
 
+        ListAppender<ILoggingEvent> listAppender = new ListAppender<>();
+        listAppender.start();
+        logger.addAppender(listAppender);
+
         this.targetAppSubmissionService.processRequestToSubmit(sdtRequestRef);
 
-        EasyMock.verify(mockIndividualRequestDao);
-        EasyMock.verify(mockConsumerGateway);
-        EasyMock.verify(mockCacheable);
-        EasyMock.verify(mockErrorMsgCacheable);
-
-        Assert.assertEquals("Bulk submission status is incorrect", IBulkSubmission.BulkRequestStatus.COMPLETED
-                .getStatus(), individualRequest.getBulkSubmission().getSubmissionStatus());
-        Assert.assertNotNull("Bulk submission completed date should be populated", individualRequest
-                .getBulkSubmission().getCompletedDate());
-        Assert.assertNotNull("Bulk submission updated date should be populated", individualRequest
-                .getBulkSubmission().getUpdatedDate());
+        List<ILoggingEvent> logList = listAppender.list;
+
+        assertFalse(logList.isEmpty());
+
+        logger.detachAndStopAllAppenders();
+        verify(mockIndividualRequestDao).persist(bulkSubmission);
+        assertEquals( IBulkSubmission.BulkRequestStatus.COMPLETED.getStatus(),
+                      individualRequest.getBulkSubmission().getSubmissionStatus(),"Bulk submission status is incorrect");
+        assertNotNull(individualRequest
+                .getBulkSubmission().getCompletedDate(),"Bulk submission completed date should be populated");
+        assertNotNull(individualRequest
+                .getBulkSubmission().getUpdatedDate(), "Bulk submission updated date should be populated");
     }
```

Comment:
May need to add code here to verify other mocked objects.

Reply:
done